### PR TITLE
Update schedule-antivirus-scans.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line abut the time zone in scheduled scans

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9958

File: microsoft-365/security/defender-endpoint/schedule-antivirus-scans.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ In addition to always-on, real-time protection and [on-demand antivirus](run-sca
 
 - If a device is unplugged and running on battery during a scheduled full scan, the scheduled scan will stop with event 1002, which states that the scan stopped before completion. Microsoft Defender Antivirus will run a full scan at the next scheduled time.
 
+- The scheduled scans execute during the local time zone of machine.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/schedule-antivirus-scans.md]:
```suggestion
- The scheduled scans execute during the local time zone of the machine.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**